config.js

```diff
@@ -15,15 +15,15 @@ module.exports = {
 	alwaysplay: false, // when set to true music will always play no matter if theres no one in voice channel.
 	debug: false, //Debug mode
 	cookieSecret: "CodingWithSudhan is epic",//Cookie Secret
-	website: "http://localhost:4200",//without the / at the end
+	website: "http://localhost:" + (this.port || "4200"), //without the / at the end
 	// You need a lavalink server for this bot to work!!!!
 	// Lavalink server; public lavalink -> https://lavalink-list.darrennathanael.com/; create one yourself -> https://darrennathanael.com/post/how-to-lavalink
 	nodes: [
 		{
-			identifier: "Main Node", //- Used for indentifier in stats commands.
-			host: "", //- The host name or IP of the lavalink server.
+			identifier: this.host || "Main Node", //- Used for identifier in stats commands.
```

Comment:
What and where `this.host` refers to?

Reply:
It refers to the same node object host propery
```js
			host: "your.lavalink.host", //- The host name or IP of the lavalink server.
```

Reply:
This is how it shows in logs when connecting to nodes. 

![image](https://user-images.githubusercontent.com/39028181/176483923-d6571d83-6b90-4d0b-8e29-c29939adb075.png)

Reply:
I thought it is defined by Heroku. The `this` keyword is very vague and you can never be 100% sure where or what it refers to. If you're not using `this` because of Heroku environment then I want it removed. What do you mean by "Heroku support" here then?

Reply:
`this` isn't vague it refers to the object where its being called, since node has an object {...} it would work. I thought there wasn't a real need for having a different identifier than host.

Also Heroku support was done via using PORT from environment variables. Heroku doesn't allow fixed ports like 4200.